src/components/search/SearchCourseCard.jsx

```diff
@@ -1,31 +1,143 @@
 import React, { useContext, useMemo } from 'react';
 import PropTypes from 'prop-types';
+import Truncate from 'react-truncate';
 import { Link } from 'react-router-dom';
+import Skeleton from 'react-loading-skeleton';
 import { AppContext } from '@edx/frontend-platform/react';
+import { camelCaseObject } from '@edx/frontend-platform/utils';
 
-const SearchCourseCard = ({ hit }) => {
+import './styles/SearchCourseCard.scss';
+
+const SearchCourseCard = ({ hit, isLoading }) => {
   const { enterpriseConfig: { slug } } = useContext(AppContext);
 
-  const courseDetails = useMemo(
+  const course = useMemo(
     () => {
-      const { title, key } = hit;
-      return { title, key };
+      if (!hit) {
+        return {};
+      }
+      return camelCaseObject(hit);
     },
     [hit],
   );
 
+  const linkToCourse = useMemo(
+    () => {
+      if (!Object.keys(course).length) {
+        return '#';
+      }
+      return `/${slug}/course/${course.key}`;
+    },
+    [isLoading, course],
+  );
+
+  const partnerDetails = useMemo(
+    () => {
+      if (!Object.keys(course).length) {
+        return {};
+      }
+
+      return {
+        primaryPartner: course.partners.length > 0 ? course.partners[0] : undefined,
+        showPartnerLogo: course.partners.length === 1,
+      };
+    },
+    [course],
+  );
+
+  // FIXME: temporarily using the course image for "How to Learn Online" until Algolia is
```

Comment:
Is there a ticket associated with this work that could be noted here?

Reply:
Not yet; I will file one to track it, though I was planning to just come back and update this outside of a specific ticket since it should be a quick change we don't really need to go through the grooming process for 🤷‍♂️ 

I will include a link to this line in that ticket.